test/system/helpers.bash

```diff
@@ -397,7 +397,7 @@ function _ensure_pod_state() {
     for i in {0..5}; do
         run_podman pod inspect $1 --format "{{.State}}"
         if [[ $output == "$2" ]]; then
-            break
+            return
         fi
         sleep 0.5
     done
```

Comment:
Please fix the fallthrough: the `is` comparison is now confusing. It should just be `die "Timed out waiting for pod $1 to enter state '$2'"`. (I thought I had requested this in the initial review of this code; obviously not. Sorry for letting that slip). Same in line 418 below.

Reply:
Thanks for checking!  We said to fix it after the initial PR has merged but I probably forgot to use `die`.